app/views/managers/show_request.html.erb

```diff
@@ -0,0 +1,33 @@
+<% @submissions.each do |submission| %>
+  <div class="container">
+    <% unless ["approved", "not approved", "reject"].include? submission[:status] %>
+      <div class="row mt-3">
+        <div class="col-1">
+          <%= submission.user_id %>
+        </div>
+        <div class="col-7">
+          <%= submission.data %>
+        </div>
+
+        <div class="col-2">
+
+        </div>
+
+        <div class="col-2">
+          <div class="row">
+            <div class="col-6 btn btn-primary btn-sm">
+              <%= link_to "Approved", set_status_manager_path(current_user.id, { status: "approved", submission: submission }), data: { turbo_method: "post" }, class: "btn" %>
```

Comment:
buttton text should be "Approve" instead of "approve" also you dont need to pass keys in hash format